examples/advanced.rs

```diff
@@ -128,15 +128,38 @@ impl egui_tiles::Behavior<Pane> for TreeBehavior {
         format!("View {}", view.nr).into()
     }
 
-    fn top_bar_rtl_ui(
+    fn top_bar_left_ui(
         &mut self,
         _tiles: &egui_tiles::Tiles<Pane>,
         ui: &mut egui::Ui,
-        tile_id: egui_tiles::TileId,
+        _tile_id: egui_tiles::TileId,
+        _tabs: &egui_tiles::Tabs,
+        _offset: f32,
+        _scroll: &mut f32,
+    ) {
+        if ui.button("⏴").clicked() {
+            *_scroll += -45.0;
```

Comment:
underscore prefix are for unused arguments to silence warnings

```suggestion
        scroll: &mut f32,
    ) {
        if ui.button("⏴").clicked() {
            *scroll += -45.0;
```